project.clj

```diff
@@ -84,7 +84,7 @@
                  ["snapshots" "http://nexus.delivery.puppetlabs.net/content/repositories/snapshots/"]]
 
   :plugins [[lein-parent "0.3.1"]
-            [puppetlabs/i18n "0.4.3"]]
+            [puppetlabs/i18n "0.5.1"]]
```

Comment:
I don't think so... and I think that, going forward, that kind of thing is hopefully supposed to be handled by the CI workflow.  But we can always update in a subsequent PR if we need to.